elixir/apps/web/lib/web/live/groups/index.ex

```diff
@@ -84,15 +87,8 @@ defmodule Web.Groups.Index do
               <div class="flex justify-center text-center text-neutral-500 p-4">
                 <div class="w-auto">
                   <div class="pb-4">
-                    No groups to display
+                    No groups to display.
                   </div>
-                  <.add_button navigate={~p"/#{@account}/groups/new"}>
```

Comment:
So this is the case why we added them, we should not have a table without an action or explanation on how to proceed in an empty state.

Reply:
I disagree. The chance a user sees an empty table and is confused on what to do next is low, and will only happen once. The remaining time they use the product, this button just gets in the way.

Reply:
Empty states of tables are a whole big deal in UX space:
https://www.toptal.com/designers/ux/empty-state-ux-design
https://www.nngroup.com/articles/empty-state-interface-design/

Reply:
So I'd say they can be links, but should not be just "oh there is nothing to see here go figure it out"

Reply:
Here is an example from one of the articles doing the same thing as we do:
<img width="913" alt="Screenshot 2023-12-21 at 11 07 56" src="https://github.com/firezone/firezone/assets/1877644/9ea5e382-34df-42ab-9e4b-124fdccadee5">

Reply:
So in all of those links, all of them solve the empty state with a single action button. We have two, which is redundant.

Reply:
And another one:
<img width="721" alt="Screenshot 2023-12-21 at 11 11 21" src="https://github.com/firezone/firezone/assets/1877644/d20e7e46-6d22-494c-be18-cc995e0b7ba8">

Reply:
The admins of our product are more technical than the average users of these consumer products, and generally don't want to be using our product UI more than they have to. So I think our design philosophy needs to be more minimalistic and get out of the way.

Reply:
It's a very far-fetched argument especially for a team size of 5 on a self-served billing or free tier, we should not be in the way of advanced users (having two actions is not staying in the way as I perceive it; and most advanced will use terraform anyways) but have a UX that is guides users and explains concepts during early stages of adoption.

Reply:
So in general I agree. But I don't think having two action buttons separated by a little whitespace and `No records to display` makes much sense. If we have more space between the two buttons, then I see your point. But they're right next to each other in each empty table.

Reply:
Here are some examples from other products in our space:

<img width="1193" alt="Screenshot 2023-12-21 at 9 31 09 AM" src="https://github.com/firezone/firezone/assets/167144/7ff577d2-08b2-4e94-ac11-307d915cd36f">


<img width="1162" alt="Screenshot 2023-12-21 at 9 31 56 AM" src="https://github.com/firezone/firezone/assets/167144/2bb43413-a9a3-4c07-9b7a-4cbe524f47fd">

<img width="1212" alt="Screenshot 2023-12-21 at 9 35 27 AM" src="https://github.com/firezone/firezone/assets/167144/7e9fb7cc-d386-41fb-bd5a-c24d52fda1e7">
<img width="1193" alt="Screenshot 2023-12-21 at 9 35 06 AM" src="https://github.com/firezone/firezone/assets/167144/9b3c14b9-7803-4957-b70e-d606cd1370ac">

Reply:
The problem is the content here, it should be not just "nothing to display" but with a summary describing what this table will show and how to proceed with it. I agree two primary-colored buttons might be too much, we can make buttons inside the table gray or turn them into links.

Another one is that here you had to move a button to "Add an IdP" to the right of the "Groups" table which doesn't feel right to me, because most users will have just one IdP, this is the action that they should take only once exactly when this table is empty. What's the point of showing it on this page after the sync is enabled? I feel it's a design decision that will be used as pattern and lead to a worse UX, specifically making it more confusing for active/advanced users.

And yeah, we have just one such place *now* but as the product grows, I bet, we have more of them and will roll this change back. Anyway, if you feel so strong about it - it's your call.

Reply:
Let's discuss more on a call, continuing here might be less productive.

Reply:
Updated to add more description action text with embedded links. Looks less busy visually IMO and still provides clear direction to the admin:

<img width="759" alt="Screenshot 2023-12-22 at 5 33 05 PM" src="https://github.com/firezone/firezone/assets/167144/bfc7adb6-9205-41f8-b22f-31642528f969">